File: Beginner/1060 (Positive Numbers)/1060.cs
using System;

namespace URI
{
    class Program
    {
        static void Main(string[] args)
        {
            int acm = 0;
            for (int i = 0; i < 6; i++)
            {
                double n = double.Parse(Console.ReadLine());
                if (n > 0)
                {
                    acm++;
                }
            }

            Console.WriteLine(acm + " valores positivos");
        }
    }
}

File: Beginner/1060 (Positive Numbers)/1060.cpp
#include <iostream>
using namespace std;

int main()
{
    double n[6],acm=0;
    for(int i=0;i<6;i++){
        cin >> n[i];
            if(n[i] >0){
            acm++;
        }

    }
    cout <<acm << " valores positivos" << endl;
    return 0;
}
